src/frontends/tensorflow_common/src/op/gelu.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_op_table.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/gelu.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+
+OutputVector translate_gelu_op(const NodeContext& node) {
+    default_op_checks(node, 1, {"GELU"});
+    auto x = node.get_input(0);
+
+    // update these lines for best translation
+    auto approximate = node.get_attribute<GeluApproximationMode>("approximate");
+    auto res = make_shared<v7::Gelu>(x, approximate);
```

Comment:
based on approximate value, you should create `approximation_mode` for `v7::Gelu`

Reply:
Done